### PR TITLE
Fix old delta robot services

### PR DESCRIPTION
**Pull request by apvanzanten**
Put the old services back in. Also rewrote large parts of them. Review thoroughly please ;)

### CODE REVIEW

File: ros/deltaRobotNode/include/DeltaRobotNode/deltaRobotNode.h [hunk 1]
```diff
@@ -90,30 +112,56 @@
 		 **/
 		Motor::MotorManager* motorManager;
 		/**
-		 * @var ros::ServiceServer moveToPointService
+		 * @var ros::ServiceServer moveToPointService_old_old
```
Comment (arjanhgroen) [inline ros/deltaRobotNode/include/DeltaRobotNode/deltaRobotNode.h]: old_old! Seems a bit strange

File: ros/deltaRobotNode/src/DeltaRobotNode.cpp [hunk 2]
```diff
@@ -120,220 +130,376 @@
...
  * 
- * @return true if the calibration was successful else false
+ * @return always true
  **/
-bool deltaRobotNodeNamespace::DeltaRobotNode::calibrate(rexosStdSrvs::Module::Request &req, rexosStdSrvs::Module::Response &res){
-	res.succeeded = false;
+bool deltaRobotNodeNamespace::DeltaRobotNode::calibrate_old(deltaRobotNode::Calibrate::Request &req, deltaRobotNode::Calibrate::Response &res){
 	if(getState() != rosMast::normal){
+		res.succeeded = false;
 		res.message = "Cannot calibrate, mast state=" + std::string(rosMast::state_txt[getState()]);
```
Comment (arjanhgroen) [inline ros/deltaRobotNode/src/DeltaRobotNode.cpp:157]: Perhaps just put this in a ROS_INFO, or both. Goes for all the services

File: ros/deltaRobotNode/include/DeltaRobotNode/Services.h [hunk 3]
```diff
@@ -33,33 +33,66 @@
...
+	
+	/**
+	 * @var const std::string CALIBRATE
+	 * Name for the service in which a deltarobot calibrates itself.
+	 **/
+	const std::string CALIBRATE = "calibrate";
+
+	// json services ----------------------------------------------------------
 	/**
 	 * @var const std::string MOVE_TO_POINT
```
Comment (KBraham) [inline ros/deltaRobotNode/include/DeltaRobotNode/Services.h]: Lui. Doe doxygen.

File: ros/deltaRobotNode/src/DeltaRobotNode.cpp [hunk 4]
```diff
@@ -120,220 +130,376 @@
...
+			}
+		}
+		
+		for(unsigned int i = 0; i < req.motion.size(); i++){	
+			ROS_INFO("moveTo: (%f, %f, %f) maxAcceleration=%f", req.motion[i].x, req.motion[i].y, req.motion[i].z, req.motion[i].maxAcceleration);
+			deltaRobot->moveTo(DataTypes::Point3D<double>(req.motion[i].x, req.motion[i].y, req.motion[i].z), req.motion[i].maxAcceleration);
+		}
 	}
-
-	res.succeeded = true;
```
Comment (KBraham) [inline ros/deltaRobotNode/src/DeltaRobotNode.cpp:222]: Moet niet weg! (Mijn app crashed dan....)

**Comment by KBraham** [comment]
seal-of-approval

**Pull request merged**